.github/workflows/ci.yml

```diff
@@ -53,7 +53,15 @@ jobs:
       - run: julia -e 'using InteractiveUtils; versioninfo(verbose=true)'
       - uses: julia-actions/cache@v1
       - uses: julia-actions/julia-buildpkg@v1
-      - uses: julia-actions/julia-runtest@v1
+      #- uses: julia-actions/julia-runtest@v1
```

Comment:
```suggestion
```

Reply:
Thanks for spotting this! I commited the suggestion in  6e7c567d73a4832fe36b44c25c2f1d1c93023852